README.md

```diff
@@ -18,8 +18,11 @@ Additional ESLint rules for Node.js
 npm install --save-dev eslint eslint-plugin-n
 ```
 
-- Requires Node.js `>=16.0.0`
-- Requires ESLint `>=7.0.0`
+| Version | Supported Node.js | Supported ESLint Version |
+|---------|-------------------|---------------------------|
+| 17.x   | `Node.js:^18.18.0 \|\| ^20.9.0 \|\| >=21.1.0`  | `ESLint>=8.23.0`          |
+| 16.x   | `Node.js>=16.0.0`   | `ESLint>=7.0.0`          |
+| 15.x   | `Node.js>=12.22.0`  | `ESLint>=7.0.0`          |
```

Comment:
**suggestion:** I think we can remove the `Node.js` and `ESLint` prefixes in this table 🤔 

```suggestion
| 17.x   | `^18.18.0 \|\| ^20.9.0 \|\| >=21.1.0`  | `>=8.23.0`          |
| 16.x   | `>=16.0.0`   | `>=7.0.0`          |
| 15.x   | `>=12.22.0`  | `>=7.0.0`          |
```